### ISSUE TITLE
mat-sed-Atst config

### ISSUE THREAD

**Issue by n2729648074** [cai525/Transformer4SED#2]
Hello, thank you very much for your contribution. 
I also reproduced the code according to your open source code. I noticed that in addition to writing "Passt" program, your mat-sed work also wrote the training code of "Atst", so I would like to know if you have done experiments on this code, and if so, could you share the config of your experiment with me?
Good luck with your research.

**Comment by cai525** [comment]
Thank you for your interest in our work! Yes, in addition to PaSST, we also experimented with ATST as an encoder. However, we found that using ATST within the MAT-SED framework yielded slightly poorer results compared to using PaSST. One possible explanation for this is that ATST employs a frame-wise encoding approach, which, while providing higher temporal resolution, may lose some frequency information that is crucial for sound event detection tasks.

As for the experimental configuration, we didn't fine-tune the parameters specifically for ATST; the parameters used are almost identical to those employed with PaSST as the encoder. We plan to upload the configuration file in the coming days. Until then, this issue will remain open.

**Comment by n2729648074** [comment]
Thank you again for your contributions. I wanted to ask you one more question about the learning rate and batch size parameter selection, because I see that you and your team have your own ideas about parameter Settings in the top meeting article. I wonder if there is any strategy in this regard?

**Comment by cai525** [comment]
> Thank you again for your contributions. I wanted to ask you one more question about the learning rate and batch size parameter selection, because I see that you and your team have your own ideas about parameter Settings in the top meeting article. I wonder if there is any strategy in this regard?
The selection of learning rate and batch size is a complex topic that depends on various factors, including the size of the dataset and the GPU memory available. Additionally, during the fine-tuning process, we use a lower learning rate for pre-trained modules (such as PaSST) because we aim to preserve  knowledge learned from the source dataset(i.e. AudioSet). Overall, I suppose that our parameter settings are not particularly unique compared to other deep learning research. I hope this answers your question.

**Comment by cai525** [comment]
>Thank you again for your contributions. I wanted to ask you one more question about the learning rate and batch size parameter selection, because I see that you and your team have your own ideas about parameter Settings in the top meeting article. I wonder if there is any strategy in this regard?

The selection of learning rate and batch size is a complex topic that depends on various factors, including the size of the dataset and the GPU memory available. Additionally, during the fine-tuning process, we use a lower learning rate for pre-trained modules (such as PaSST) because we aim to preserve knowledge learned from the source dataset(i.e. AudioSet). Overall, I suppose that our parameter settings are not particularly unique compared to other deep learning research. I hope this answers your question.

**Comment by cai525** [comment]
The configuration files of training MAT-SED with ATST as a backbone have been added, and you can run this [shell script](https://github.com/cai525/Transformer4SED/blob/main/exps/mat-sed/atst/) to start training.

**Comment by cai525** [comment]
> Thank you very much for taking the time to upload the configuration file. I noticed that you did not use filt_aug for data augmentation in atst, but replaced it with another one. Is that because the augmentation does not apply to atst? Thank you very much for your answer.

The normalization method in ATST([link](https://github.com/cai525/Transformer4SED/blob/main/src/models/atst/atst_feature_extraction.py)) is a little different from the method in the implementation of filt_aug, so we don't use filt_aug in the implementation of atst.  Of course you can use filt_aug here but the related code about feature extraction needs to be changed.  I hope this answers your question.

**Comment by n2729648074** [comment]
Thank you very much for your answer, which is very helpful to me. I wish you every success in your scientific research.
